src/urllib3/connectionpool.py

```diff
@@ -414,18 +397,7 @@ def _make_request(
 
         # Receive the response from the server
         try:
-            try:
```

Comment:
Woo, this is super great and I've been waiting to remove this since day 1 :tada: